js/viewer/viewer.js

```diff
@@ -90,7 +90,18 @@ var odfViewer = {
 		}
 
 		OC.addStyle('richdocuments', 'mobile');
-		var $iframe = $('<iframe id="richdocumentsframe" scrolling="no" allowfullscreen style="width:100%;height:100%;display:block;position:absolute;top:0;z-index:60;" src="'+viewer+'" />');
+		var $button = $('<div class="richdocuments-sharing"><a class="icon-shared icon-white"></a></div>');
```

Comment:
Ah maybe we need a check here to see the header is actually present etc?
Because in the mobile UI this won't be there for example.

Reply:
I'd guess the header would just get hidden then on mobile so it should still be fine. Or would there be a different route that renders a template without the header?

Reply:
It is a different route that does not load the header at all.

Reply:
Ok, I'll make it optional then, when a header is present.